### PR TITLE
use translation keys for admin group names

### PR DESCRIPTION
**Pull request by core23**
Removed duplicate translation for ``CommentAdmin`` groups and renamed the ``PostAdmin`` group labels.

### CODE REVIEW

**Comment by OskarStark** [comment]
I think we should change them to something like post_label ...

**Comment by core23** [comment]
Renamed the group labels in the ``PostAdmin``. 

Should I add a BC hint for it?

**Comment by OskarStark** [comment]
@Soullivaneuh IMO, this is not a BC, what do you say?

@core23 thank you! :+1: , please rebase

**Comment by OskarStark** [comment]
@core23 please add the english translations directly, thank you!

**Comment by core23** [comment]
Ready for review

**Comment by OskarStark** [comment]
Please rename commit ``use translation keys for admin group names``

**Comment by OskarStark** [comment]
Ping @Soullivaneuh can you please translate the french file?

@sagikazarmark can you help with your language?

**Comment by sagikazarmark** [comment]
Sure.

``` xml
            <trans-unit id="group_post">
                <source>group_post</source>
                <target>Bejegyzés</target>
            </trans-unit>
            <trans-unit id="group_status">
                <source>group_status</source>
                <target>Státusz</target>
            </trans-unit>
            <trans-unit id="group_classification">
                <source>group_classification</source>
                <target>Osztályozás</target>
            </trans-unit>
```

**Comment by OskarStark** [comment]
Thank you

**Comment by OskarStark** [comment]
@core23 after adding the translations by @sagikazarmark we can merge your PR

File: Resources/translations/SonataNewsBundle.es.xliff [hunk 1]
```diff
@@ -415,6 +415,18 @@ Modera el comentario de forma rápida :
                 <source>show.label_tags</source>
                 <target>show_label_tags</target>
             </trans-unit>
+            <trans-unit id="group_post">
+                <source>group_post</source>
+                <target>group_post</target>
```
Comment (Soullivaneuh) [inline Resources/translations/SonataNewsBundle.es.xliff]: Article

File: Resources/translations/SonataNewsBundle.es.xliff [hunk 2]
```diff
@@ -415,6 +415,18 @@ Modera el comentario de forma rápida :
...
+            </trans-unit>
+            <trans-unit id="group_status">
+                <source>group_status</source>
+                <target>group_status</target>
```
Comment (Soullivaneuh) [inline Resources/translations/SonataNewsBundle.es.xliff]: Statut

File: Resources/translations/SonataNewsBundle.es.xliff [hunk 3]
```diff
@@ -415,6 +415,18 @@ Modera el comentario de forma rápida :
...
+            </trans-unit>
+            <trans-unit id="group_classification">
+                <source>group_classification</source>
+                <target>group_classification</target>
```
Comment (Soullivaneuh) [inline Resources/translations/SonataNewsBundle.es.xliff]: Classification

File: Resources/translations/SonataNewsBundle.es.xliff (hunk 1 above)
Comment (OskarStark) [inline Resources/translations/SonataNewsBundle.es.xliff]: You are speaking Spanish as well? oO

Comment (Soullivaneuh) [inline Resources/translations/SonataNewsBundle.es.xliff]: Oops. Wrong file. It's for FR. :-)

**Comment by core23** [comment]
Added French, Hungarian and Spanish translations

**Comment by core23** [comment]
@OskarStark Ready to merge :)

**Comment by OskarStark** [comment]
Only CS Fixer fails because of: FriendsOfPHP/PHP-CS-Fixer#1450

But the other things are green, so i merge this PR as it is

Thank you @core23

**Pull request merged**